spec/features/atom/dashboard_spec.rb

```diff
@@ -7,7 +7,7 @@
     context "projects atom feed via private token" do
       it "should render projects atom feed" do
         visit dashboard_path(:atom, private_token: user.private_token)
-        body.should have_selector("feed title")
+        expect(body).to have_selector("feed title")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.